src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -1048,6 +1048,12 @@ public string[] LiteralPath
         [Parameter]
         public SwitchParameter List { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value that when true, causes only the matching part of the string to be returned.
```

Comment:
Worth noting that this causes a `string` rather than a `MatchInfo` to be returned